### ISSUE TITLE
Ability to re-convert to Mp3

### ISSUE THREAD

**Issue by frank392a** [rmcrackan/Libation#926]
Please consider adding the feature to re-convert to MP3s. This would be helpful for someone who has converted their entire library to MP3 only to realize later that converting to a lower quality could save hard disk space.

Thank you so very much!!!

**Comment by rmcrackan** [comment]
Good idea. For now, you can change to your desired setting then re-download. To re-download a single book, right click. For the whole library, go to Visible books and manually set download status to 'not downloaded'. On your next download, all books with a red stop light will be re-downloaded with your new settings.

**Comment by CLHatch** [comment]
> Good idea. For now, you can change to your desired setting then re-download. To re-download a single book, right click. For the whole library, go to Visible books and manually set download status to 'not downloaded'. On your next download, all books with a red stop light will be re-downloaded with your new settings.

I'm not sure I've tried it, but I think I read previously that when doing this, the old files remain, don't they?  That could be an issue, say if you're changed whether or not it outputs chapters as individual files.

**Comment by rmcrackan** [comment]
@CLHatch Correct. If you don't want the old files, you'll have to delete those yourself, preferably before doing a new download.

Libation generates new files but it will not move, name, or remove files, including those created by Libation. Not now; not ever. The edge cases are legion and no 2 people would agree on the solutions.

**Comment by donburch888** [comment]
In my case I was trying to work out the naming template options, and hoped that re-downloading the same small file would re-do the file naming - but that didn't happen. 

I worked around it by (a) download different (increasingly larger) files (thank goodness for all the free short stories) and (b) reading the instructions several times until I worked out where to place the minus signs.  So ... not a real problem, but if you do revisit this request you might also consider an option to reapply the file naming.

**Comment by CLHatch** [comment]
> In my case I was trying to work out the naming template options, and hoped that re-downloading the same small file would re-do the file naming - but that didn't happen.
> 
> I worked around it by (a) download different (increasingly larger) files (thank goodness for all the free short stories) and (b) reading the instructions several times until I worked out where to place the minus signs. So ... not a real problem, but if you do revisit this request you might also consider an option to reapply the file naming.

There are actually was an option added a while back that lets you see exactly how the files for book will be named.  I'm not at the PC currently, but I believe you right-click on the book and select "Edit Folder Template" or "Edit File Template". It will bring you to the settings page for the naming template. The difference is, the example output shown will be for the book you right-clicked on instead of the normal "Sherlock Holmes" example.

**Comment by rmcrackan** [comment]
This is a fine request but after all this time, clearly I'm not actually going to get around to it. I think it's time for me to just admit defeat.
